web/src/stories/WeatherMap.stories.tsx

```diff
@@ -15,7 +16,7 @@ type PropTypes = React.ComponentProps<typeof WeatherMap>
 const Template: Story<PropTypes> = () => (
   <div style={{ height: '500px' }}>
     <Provider store={store}>
-      <WeatherMap />
+      {<WeatherMap center={CENTER_OF_BC} isCollapsed={false} setMapCenter={_ => {}} />}
```

Comment:
Oh - actually - could you please resolve the sonarcloud code smell associated with this?